src/main/java/programminglife/ProgrammingLife.java

```diff
@@ -54,8 +54,8 @@ public void start(Stage stage) throws IOException {
         close.setOnAction(event -> primaryStage.fireEvent(
                 new WindowEvent(primaryStage, WindowEvent.WINDOW_CLOSE_REQUEST))
         );
-        primaryStage.setMinWidth(600);
-        primaryStage.setMinHeight(400);
+        primaryStage.setMinWidth(1280);
+        primaryStage.setMinHeight(720);
```

Comment:
I am not a fan of this. On my 2012 MBP (1280x720) this would cause problems if status bar+dock is higher than 80 px.